src/Discord.Net.Core/Discord.Net.Core.csproj

```diff
@@ -1,14 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../../Discord.Net.targets" />
-  <PropertyGroup>
-    <AssemblyName>Discord.Net.Core</AssemblyName>
-    <RootNamespace>Discord</RootNamespace>
-    <Description>The core components for the Discord.Net library.</Description>
-    <TargetFrameworks>net45;netstandard1.1;netstandard1.3</TargetFrameworks>
-  </PropertyGroup>
+  <Choose>
+    <When Condition=" '$(OS)' == 'Windows_NT' ">
+      <PropertyGroup>
+        <AssemblyName>Discord.Net.Core</AssemblyName>
+        <RootNamespace>Discord</RootNamespace>
+        <Description>The core components for the Discord.Net library.</Description>
+        <TargetFrameworks>net45;netstandard1.1;netstandard1.3</TargetFrameworks>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <AssemblyName>Discord.Net.Core</AssemblyName>
+        <RootNamespace>Discord</RootNamespace>
+        <Description>The core components for the Discord.Net library.</Description>
+        <TargetFrameworks>netstandard1.1;netstandard1.3</TargetFrameworks>
```

Comment:
Couldn't we just change the `TargetFramework` and leave the rest out of the condition? You're duplicating the `AssemblyName` and such and isn't very clean.

Reply:
Yes, we should be able to just use something like
```xml
  <PropertyGroup>
    <AssemblyName>Discord.Net.Core</AssemblyName>
    <RootNamespace>Discord</RootNamespace>
    <Description>The core components for the Discord.Net library.</Description>
    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net45;netstandard1.1;netstandard1.3</TargetFrameworks>
    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.1;netstandard1.3</TargetFrameworks>
  </PropertyGroup>
```

Reply:
@Still34 Fixed in 319bbcffad16e856fd23ffa19ea1c93b9324af7b